web-jehlomat/src/Components/Inputs/SelectInput/SelectInput.tsx

```diff
@@ -1,9 +1,38 @@
+import styled from '@emotion/styled';
+import { Select, MenuItem, FormHelperText, FormControl } from '@mui/material';
 import { FC } from 'react';
+import { Label } from '../shared';
 
-interface SelectInput {}
+interface IProps {
+    name: string;
+    label: string;
+    values: any[];
```

Comment:
You already know, that values are object with Key and Name, why don't specify it right away? Also, the key is type of number (or null), because the value is that type.